app/models/mixins/miq_provision_quota_mixin.rb

```diff
@@ -250,40 +261,88 @@ def quota_find_active_prov_request_by_owner(options)
     quota_find_active_prov_request(options).select { |p| email.casecmp(p.get_option(:owner_email).to_s.strip) == 0 }
   end
 
+  def quota_find_active_prov_request_by_user(options)
+    quota_find_active_prov_request(options).select do |p|
```

Comment:
@bdunne The method needs to have the options argument because it's called by the quota_provision_stats method which calls all of the provision methods with options:
def quota_provision_stats(prov_method, options) 
....
send(prov_method, options).each do |pr|